marlowe-playground-server/app/PSGenerator.hs

```diff
@@ -170,9 +170,9 @@ writeUsecases outputDir = do
          <> multilineString "escrow" escrow
          <> multilineString "zeroCouponBond" zeroCouponBond
          <> multilineString "couponBondGuaranteed" couponBondGuaranteed
-        usecasesModule = psModule "Meadow.Contracts" usecases
-    createDirectoryIfMissing True (outputDir </> "Meadow")
-    BS.writeFile (outputDir </> "Meadow" </> "Contracts.purs") usecasesModule
+        usecasesModule = psModule "Haskell.Contracts" usecases
```

Comment:
`Marlowe.Contracts`?

Reply:
`Marlowe.Contracts` already exists and contains string representations of Marlowe code, this module contains string representations of Haskell code